CHANGELOG.md

```diff
@@ -29,21 +32,74 @@
 
 - [#1781](https://github.com/tigerbeetle/tigerbeetle/pull/1781),
   [#1784](https://github.com/tigerbeetle/tigerbeetle/pull/1784),
-  [#1765](https://github.com/tigerbeetle/tigerbeetle/pull/1765)
+  [#1765](https://github.com/tigerbeetle/tigerbeetle/pull/1765),
+  [#1816](https://github.com/tigerbeetle/tigerbeetle/pull/1816),
+  [#1808](https://github.com/tigerbeetle/tigerbeetle/pull/1808),
+  [#1802](https://github.com/tigerbeetle/tigerbeetle/pull/1802),
+  [#1798](https://github.com/tigerbeetle/tigerbeetle/pull/1798),
+  [#1793](https://github.com/tigerbeetle/tigerbeetle/pull/1793),
+  [#1805](https://github.com/tigerbeetle/tigerbeetle/pull/1805)
 
   Documentation improvements.
 
+- [#1813](https://github.com/tigerbeetle/tigerbeetle/pull/1813)
+
+  Improve Docker experience by handling `SIGTERM` through [tini](https://github.com/krallin/tini).
+
+- [#1800](https://github.com/tigerbeetle/tigerbeetle/pull/1800)
+
+  For reproducible benchmarks, allow setting `--seed` on the CLI.
+
 ### Internals
 
 - [#1640](https://github.com/tigerbeetle/tigerbeetle/pull/1640),
-  [#1782](https://github.com/tigerbeetle/tigerbeetle/pull/1782)
+  [#1782](https://github.com/tigerbeetle/tigerbeetle/pull/1782),
+  [#1788](https://github.com/tigerbeetle/tigerbeetle/pull/1788)
 
   Move `request_queue` outside of `vsr.Client`.
 
 - [#1775](https://github.com/tigerbeetle/tigerbeetle/pull/1775)
 
   Extract `CompactionPipeline` to a dedicated function.
 
+- [#1797](https://github.com/tigerbeetle/tigerbeetle/pull/1797)
+
+  Use a FIFO as a block_pool instead of trying to slice arrays during compaction.
+
```

Comment:
This was a bugfix (the previous version could crash) so this could move up to the Safety/Performance section.